change/@fluentui-react-motion-80dc5fe2-1c39-41d8-921f-3e5d78554267.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33357&commitId=e1c4fdc6d6f81d9b22479049ec692653587f8712&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>